pkg/sql/sqltestutils/telemetry.go

```diff
@@ -366,7 +361,9 @@ func formatSQLStats(stats []appstatspb.CollectedStatementStatistics) string {
 	for i := range stats {
 		s := &stats[i]
 
-		if strings.HasPrefix(s.Key.App, catconstants.InternalAppNamePrefix) {
+		if strings.HasPrefix(s.Key.App,
+			catconstants.InternalAppNamePrefix) || strings.HasPrefix(s.Key.App,
+			catconstants.DelegatedAppNamePrefix) {
```

Comment:
Seeing this makes me wonder if we should have some helpers that combine some of these checks together. Internal + Delegated are commonly together for instance.